packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/LightCurveViewer/LightCurveViewerContainer.js

```diff
@@ -94,7 +94,7 @@ export function LightCurveViewerContainer({
     dataPoints = zip(JSONdata.x, JSONdata.y)
   }
 
-  if (!subject.id) {
+  if (!subject.id || !dataPoints.length) {
```

Comment:
Do we want to check for both using `&&` instead of `||`?
If the answer is `||`, is the priority correct? Meaning, does it matter what is check first (`subject.id` or `dataPoints.length`)?

Reply:
I think it's ok like this because if there's no subject ID (`!subject.id`) / i.e. first condition is `true`, we don't care about if there is are any dataPoints* and would definitely want it to return `null`. But if there is a subject ID / first condition `false`, then we want to check for `dataPoints`, and if there aren't any `dataPoints` (length is 0), then we also want to return `null`.

`dataPoints` won't (really really shouldn't) have a length if there's no subject, as it's defined by the JSON data associated with the subject, so if `dataPoints` has a length but there's no subject ID then something very wrong is going on.